### PR TITLE
Issue 5260 - BUG - OpenLDAP allows multiple names of memberof overlay

### LINKED ISSUE #5260: OpenLDAP changes OC of memberof overlay
I think openldap allows two different objectclass names for the memberOf overlay. This causes the overlay to be mis-detected as unknown and prevents the upgrade (but it does correctly flag as an unknown overlay in the checklist).

### PR DESCRIPTION
**Pull request by Firstyear**
Bug Description: Openldap allowed multiple names of the memberof overlay
which mean that in some cases we wouldn't migrate the plugin.

Fix Description: Accept both objectclass names.

fixes: https://github.com/389ds/389-ds-base/issues/5260

Author: William Brown <william@blackhats.net.au>

Review by: ???

### CODE REVIEW

File: dirsrvtests/tests/suites/openldap_2_389/migrate_memberof_test.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 William Brown <william@blackhats.net.au>
```
Comment (droideck) [inline dirsrvtests/tests/suites/openldap_2_389/migrate_memberof_test.py]: Don't live in the past, mate :)

**Review by droideck — APPROVED** [review]
The rest looks good to me!

File: dirsrvtests/tests/suites/openldap_2_389/migrate_memberof_test.py (hunk 1 above)
Comment (Firstyear) [reply dirsrvtests/tests/suites/openldap_2_389/migrate_memberof_test.py]: I can live whenever I want!

**Comment by Firstyear** [comment]
```
   9c6cc3014..d50340ba5  389-ds-base-2.1 -> 389-ds-base-2.1
   670d4cc14..4d2d010b8  389-ds-base-2.0 -> 389-ds-base-2.0
```

**Pull request merged**